Gemfile

```diff
@@ -89,13 +94,15 @@ group :development do
 end
 
 group :test do
-  gem 'capybara'
   gem 'factory_bot_rails'
   gem 'rspec-rails'
   gem 'simplecov'
 end
 
 group :ui do
   gem 'capybara-selenium'
+  gem 'rspec'
```

Comment:
This isn't needed because the ui group is added on top of test, which got rspec via rspec-rails.

Reply:
removed